nilearn/tests/test_signal.py

```diff
@@ -831,6 +845,11 @@ def test_clean_zscore():
     np.testing.assert_almost_equal(cleaned_signals.mean(0), 0)
     np.testing.assert_almost_equal(cleaned_signals.std(0), 1)
 
+    # Repeating test above but for new correct strategy
+    cleaned_signals = clean(signals, standardize='zscore_sample')
+    np.testing.assert_almost_equal(cleaned_signals.mean(0), 0)
+    np.testing.assert_almost_equal(cleaned_signals.std(0), 1, decimal=3)
```

Comment:
the result is thus unchangde with the new standardization (line 851 == line 846)

Reply:
@bthirion, the results are changed slightly so I has to lower the precision hence why I set the decimal to `3` places instead of the default `7` it was before (comment https://github.com/nilearn/nilearn/pull/3474#issuecomment-1412098490). I'm not sure if this is expected or not

Reply:
Yes, but I was actually noting that the two approaches yield the same results. Do we have a test where the two ways of standardizing yield a distinct outcome ?

Reply:
Ah ok got it. Will add that